.github/workflows/medic-benchmarks.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       cluster: zeebe-cluster
       cluster-region: europe-west1-b
       ref: ${{ needs.benchmark-data.outputs.ref }}
-      bechmark-load: >
+      benchmark-load: >
```

Comment:
:thinking: We have the same typo in the benchmark workflow: https://github.com/camunda/zeebe/blob/main/.github/workflows/benchmark.yml#L20

We need to rename both.

Reply:
Looks like it works now: https://github.com/camunda/zeebe/actions/runs/3618326752 :sweat_smile: